support-frontend/assets/pages/paper-subscription-checkout/components/paperCheckoutForm.jsx

```diff
@@ -195,7 +195,8 @@ function PaperCheckoutForm(props: PropTypes) {
   const [digiSubPriceString, setDigiSubPriceString] = useState<string>('');
   const [includesDigiSub, setIncludesDigiSub] = useState<boolean>(false);
   const simplePrice = digiSubPriceString.replace(/\/(.*)/, ''); // removes anything after the /
-  const cleanedPrice = simplePrice.replace(/\.(.*)/, ''); // removes decimal point if there is one
+  const priceHasRedundantFloat = simplePrice.split('.')[1] === '00'; // checks whether price is something like '£10.00'
```

Comment:
Although that does add a bit of complexity so happy to leave it out if you prefer

Reply:
I agree that this would be better - the problem is that this is just about presentation. The price that flows in is the price that will be added to the total in the order summary, and we need it to match what we present. I think if we want to validate the price it should be happening higher up the food chain, shouldn't it?

My understanding is that promotion discounts are applied to the digital add-on, so this is a factor that's a bit out of our control, and could result it some strange-looking (but correct) prices, like £1.03. 

I'm very happy to discuss this if I've misunderstood what you meant, however.